script/delete_weird_company.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../config/environment',  __FILE__)
+
+weird_company_wql = {
+  "not"=>{"type"=>"Company"},
+  "left_plus"=>[
+    {"type"=>"Metric"},
+    {"right_plus"=>[{"type"=>"Year"},{"type"=>"Metric Value"}]}
+  ]
+}
+Card::Auth.as_bot
+weird_companies = Card.search weird_company_wql
+weird_companies.each do |company|
+  related_cards_wql = {:left=>{:left=>{:type_id=>Card::MetricID},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.